extensions/wikia/Email/Controller/WeeklyDigestController.class.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Email\Controller;
+
+use \Email\EmailController;
+
+class WeeklyDigestController extends EmailController {
+
+	const LAYOUT_CSS = 'weeklyDigest.css';
+
+	protected $digestData;
+
+	protected function getSubject() {
+		return $this->getMessage( 'emailext-weeklydigest-subject' )->text();
+	}
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-weeklydigest-summary' )->text();
+	}
+
+	public function initEmail() {
+		$this->digestData = $this->request->getVal( 'digestData' );
```

Comment:
Two related notes; 1) I don't see a ```getEmailSpecificFormFields``` method here 2) The ```digestData``` parameter as is will be impossible to pass from the email tool :(

I don't have any super clever ideas, but maybe we could test whether ```digestData``` is a string or an array, and if its a string assume is JSON and decode it into an array. Then at least we could pass ```digestData``` via the tool, even if it is super awkward.